clients/src/main/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManager.java

```diff
@@ -116,14 +116,14 @@ public class HeartbeatRequestManager implements RequestManager {
     private final HeartbeatMetricsManager metricsManager;
 
     public HeartbeatRequestManager(
-        final LogContext logContext,
-        final Time time,
-        final ConsumerConfig config,
-        final CoordinatorRequestManager coordinatorRequestManager,
-        final SubscriptionState subscriptions,
-        final MembershipManager membershipManager,
-        final BackgroundEventHandler backgroundEventHandler,
-        final Metrics metrics) {
+            final LogContext logContext,
+            final Time time,
```

Comment:
Please don't submit white-space only changes unless you are otherwise editing that fragment of the code.

Reply:
Fixed.